src/components/20-molecules/input-phone/index.react.d.ts

```diff
@@ -5,6 +5,8 @@ export interface AXAInputPhoneProps {
   errorprefix: string;
   lang?: string;
   defaultarea?: string;
+  value?: Object;
+  onChange?: () => void;
```

Comment:
onChange handler should accept an event argument, so that the consument can read value:

https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/src/components/10-atoms/input-text/index.react.d.ts#L28